src/logger.c

```diff
@@ -11,7 +11,7 @@
 LoggerData g_logger;
 
 #ifndef min
-#define	min(a,b)	(((a) < (b)) ? (a) : (b))
+#define min(a,b) (((a) < (b)) ? (a) : (b))
```

Comment:
min is defined in microchip's stdlib.h, but it's non standard and doesn't exist in 'normal' stdlib.h.

So I can put the #ifndef in a header say 'min.h' which then includes stdlib.h and defines a min if min isn't already defined. The catch there is that people working on the microchip side need to remember to include min.h instead of stdlib.h even though the latter works anyway.

I figured it's more explicit about what's going on if the #ifndef is directly in the .c .